src/components/Card/Card.scss

```diff
@@ -17,18 +20,24 @@
 }
 
 .Header {
-  padding: spacing() spacing() 0;
+  padding-top: spacing();
+  padding-right: spacing();
+  padding-left: spacing();
+  padding-bottom: $header-spacing;
 
```

Comment:
Is there a reason this isn't using `padding`?

```suggestion
padding: spacing() spacing() $header-spacing spacing();
```

Reply:
Ryan commented to split out the other larger screen view to make it easier to understand which paddings had changed between the two views. I shouldn't have changed the mobile view one though so I have changed those back.